package.json

```diff
@@ -1,26 +1,36 @@
 {
   "name": "replay-cli",
-  "version": "1.0.1",
-  "description": "",
-  "main": "index.js",
+  "private": true,
+  "packageManager": "yarn@4.1.1",
+  "workspaces": [
+    "packages/*"
+  ],
   "directories": {
     "example": "examples"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "test": "npm run test:unit",
-    "test:unit": "lerna exec npm run test",
-    "typecheck": "lerna exec npm run typecheck",
-    "lint": "prettier --check ."
+    "bootstrap": "yarn workspaces foreach --all --topological run build",
+    "test": "yarn run test:unit",
+    "test:unit": "yarn workspaces foreach --all --exclude=. --topological run test",
+    "typecheck": "yarn workspaces foreach --all run typecheck",
+    "lint": "prettier --check .",
+    "release": "yarn run bootstrap && yarn workspaces foreach --all --exclude=. --topological npm publish --tolerate-republish"
   },
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "lerna": "^4.0.0",
+    "@replayio/playwright": "^1.1.16",
+    "@replayio/replay": "^0.21.4",
```

Comment:
Yes-ish. That's what makes `npx` calls in [Earthfile](https://github.com/replayio/replay-cli/blob/ca72e926c901f6b3f3199bdac5670f260bcc5b3f/Earthfile) to work. Alternatively, we could call those bin scripts using relative (in-monorepo) paths instead of `npx`. I assume the goal is to call the local versions of those and not the ones that are published